doc/cephfs/fs-volumes.rst

```diff
@@ -3,11 +3,12 @@
 FS volumes and subvolumes
 =========================
 
-A  single source of truth for CephFS exports is implemented in the volumes
-module of the :term:`Ceph Manager` daemon (ceph-mgr). The OpenStack shared
-file system service (manila_), Ceph Container Storage Interface (CSI_),
+The volumes
+module of the :term:`Ceph Manager` daemon (ceph-mgr) provides a single
+source of truth for CephFS exports . The OpenStack shared
```

Comment:
@anthonyeleven - There is an extraneous space that snuck in before the period.

Reply:
Thanks, pushing the fix.